pkg/conference/data_channel_message_processor.go

```diff
@@ -28,47 +36,61 @@ func (c *Conference) processSelectDCMessage(participant *Participant, msg event.
 		}
 	}
 
-	// Subscribe to the found tracks.
+	// Subscribe to the found tracks
 	for _, track := range tracks {
+		participant.logger.WithField("track_id", track.ID()).Debug("Subscribing to track")
 		if err := participant.peer.SubscribeTo(track); err != nil {
 			participant.logger.Errorf("Failed to subscribe to track: %v", err)
 			return
 		}
 	}
-}
-
-func (c *Conference) processAnswerDCMessage(participant *Participant, msg event.SFUMessage) {
-	participant.logger.Info("Received SDP answer over DC")
 
-	if err := participant.peer.ProcessSDPAnswer(msg.SDP); err != nil {
-		participant.logger.Errorf("Failed to set SDP answer: %v", err)
-		return
-	}
+	// TODO: Handle unsubscribe
```

Comment:
Ah, so we don't want to implement it right now? Could we probably move this TODO a bit higher in a function (maybe before handling the `subscribe`?), so that we don't forget about it (it would also influence the `subscribe` handling when e.g. `unsubscribe` would contain tracks that are also mentioned in `subscribe` which is a bug on a client side, but must be handled by the server properly)

Reply:
> Ah, so we don't want to implement it right now?

It didn't feel like this should be a part of this PR. Will do a follow-up

> Could we probably move this TODO a bit higher in a function (maybe before handling the `subscribe`?), so that we don't forget about it (it would also influence the `subscribe` handling when e.g. `unsubscribe` would contain tracks that are also mentioned in `subscribe` which is a bug on a client side, but must be handled by the server properly)

Will do